### DISCUSSION TITLE
[Help] How to add task result into logging file

### DISCUSSION THREAD

**Discussion by xiaopeng163** [nornir-automation/nornir#730] [Q&A]
for example, I have a task to execute one show command, and I got logging like below

```
2021-10-12 22:54:23,675 - nornir.core.task -    DEBUG -      start() - Host 'r1249': running task '[pre-check] - show run interface GigabitEthernet2/8'
```

from print_result, I got the output like below

```
---- [pre-check] - show run interface GigabitEthernet2/8 ** changed : False ---- INFO
Building configuration...

Current configuration : 217 bytes
!
interface GigabitEthernet2/8
 description peng xiao test
 no ip address
 logging event link-status
 shutdown
 udld port
 no mdix auto
 no cdp enable
 spanning-tree portfast edge
 spanning-tree bpduguard enable
end
```

so how can I add the print_result output into the logging file?

**Comment by dbarrosop** [comment]
I am afraid `print_result` doesn't support it so you have a couple of options:

1. Modify `print_result` so it is supported (shouldn't be too hard, it just needs a `file` argument that can be passed to `print()`).
2. Capture sys.stdout and redirect it to a file

Feel free to open a PR if you decide to go with (1)
